bpython/_internal.py

```diff
@@ -21,7 +21,7 @@ def __repr__(self) -> str:
             "or help(object) for help about object."
         )
 
-    def __call__(self, *args, **kwargs) -> None:
+    def __call__(self, *args: str, **kwargs: int) -> None:
```

Comment:
why are kwargs typed as int, is this based on the call signature of a helper? Maybe we should change the signature to match the helper object.